src/scripts/irregularUtils.js

```diff
@@ -20,7 +20,8 @@ function flattenPages(data) { // function to flatten out the paginated qHyperCub
   return flat;
 }
 
-function pageExtensionData(me, $el, layout, callback, ref, maxDataPages) { //(this, extension DOM element, layout object from Sense, your callback)
+// (this, extension DOM element, layout object from Sense, your callback)
+export function pageExtensionData(me, $el, layout, callback, ref, maxDataPages) {
```

Comment:
Here you use `export` of the function where it's defined, but in `lasso_adj.js` you use `export default setupLasso;` at the end of the file. I suggest we go one of these patterns to be consistent. Pick your favourite :)

Reply:
Good catch! I think i forgot to remove the default one once finding that this was the only function being used, i'll take a look at it.